debian/changelog

```diff
@@ -1,4 +1,4 @@
-sb-vision (0) unstable; urgency=low
+sb-vision (2018.2.0) unstable; urgency=low
```

Comment:
I'm pretty sure we should _add_ to the entries in this file, not replace them.

Reply:
Good point